### DISCUSSION TITLE
Mend Renovate App is scheduled for hourly running

### DISCUSSION THREAD

**Discussion by devsheva** [renovatebot/renovate#31106] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I noticed that renovate keeps running hourly on the repo i configured, and I can't find a way to change this behaviour even if I've scheduled it to run only on weekends

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
The Mend bot will run whenever Mend decides. You can control when it makes updates by using `schedule` in your repo config and you may also be interested in setting `updateNotScheduled=true` if you don't want any rebases outside of schedule

**Reply by devsheva** [reply]
I already have a schedule in my repo config, but I didn't know about the setting. This might be very good. In my opinion it should be more visible in the docs, cause I think someone that sees the same behaviour I encountered will end to ask itself the same question, but this is just a suggestion

**Reply by rarkins** [reply]
You can always submit PRs improving the docs
